bpf/kmesh/ads/include/local_ratelimit.h

```diff
@@ -0,0 +1,195 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
+/* Copyright Authors of Kmesh */
+
+#ifndef __KMESH_LOCAL_RATE_LIMIT_H__
+#define __KMESH_LOCAL_RATE_LIMIT_H__
+
+#include "bpf_log.h"
+#include "bpf_common.h"
+#include "kmesh_common.h"
+#include "listener/listener.pb-c.h"
+
+struct ratelimit_key {
+    union {
+        struct {
+            __u32 ipv4;   /* Destination IPv4 address. */
+            __u32 port;   /* Destination port. */
+            __u32 family; /* Address family (e.g., AF_INET) */
```

Comment:
So is this node level ratelimit, compared to sidecar, it is on all node local clients requeests

Reply:
yes, global ratelimit needs a external service, i think it is not feasible for ebpf.